pkg/app/cockroachdb.go

```diff
@@ -57,7 +57,7 @@ func (c *CockroachDB) Init(context.Context) error {
 	return err
 }
 
-func (c *CockroachDB) Install(ctx context.Context, namespace string) error {
+func (c *CockroachDB) Install(ctx context.Context, namespace string) error { //nolint:dupl // Not a duplicate, common code already extracted
```

Comment:
This is strange. Why the check is failing for `Install()` but not other methods like `IsReady()`, `Uninstall()`

Reply:
Not so sure, we still do have common code, but logs are different mostly and it didn't make sense to move them to a separate utility.

Reply:
I suppose it's not taking comments into consideration.